README.md

```diff
@@ -4,6 +4,10 @@ type: evaluation
 submission_name: MTEB
 ---
 
+> [!NOTE]  
+> Previously it was possible to submit models results to MTEB by adding the results to the model metadata. This is no longer an option as we want to ensure high quality metadata.
```

Comment:
Question: is that currently enforced? Or will only be in effect once the new leaderboard goes public?

Reply:
pr. #57 removed these